app/models/in_person_enrollment.rb

```diff
@@ -3,6 +3,9 @@
 class InPersonEnrollment < ApplicationRecord
   belongs_to :user
   belongs_to :profile
+  # rubocop:disable Rails/InverseOf
+  belongs_to :service_provider, foreign_key: 'issuer', primary_key: 'issuer', optional: true
+  # rubocop:enable Rails/InverseOf
```

Comment:
Note to self: Check if we need to bypass Rubocop here. Just followed common practice for other `belongs_to`, but presumably there's a reason the cop exists.

Reply:
might be useful for `user.in_person_enrollments` as the inverse 🤷

Reply:
As [discussed below](https://github.com/18F/identity-idp/pull/6707#discussion_r942915262), I ended up setting up the inverses since it seems to be a good practice without much downside that I can tell.